src/main/homework/day8/Counter.java

```diff
@@ -0,0 +1,26 @@
+package main.homework.day8;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Counter {
+    public static void main(String[] args) throws IOException {
+        Counter counter = new Counter();
+        counter.counter();
```

Comment:
Maybe it's better to change method or object name for better understanding? 
To avoid code like:
`Counter counter = new Counter();
        counter.counter();`